sys/net/routing/nhdp/nhdp.h

```diff
@@ -0,0 +1,207 @@
+/*
+ * Copyright (C) 2014 Freie Universität Berlin
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @defgroup    nhdp NHDP
+ * @ingroup     net
+ * @brief       The MANET Neighborhood Discovery Protocol (RFC 6130)
+ * @{
+ *
+ * @file
+ * @brief       Interface for core functionality of NHDP
+ *
+ * @author      Fabian Nack <nack@inf.fu-berlin.de>
+ */
+
+#ifndef NHDP_H_
+#define NHDP_H_
+
+#include "timex.h"
+#include "kernel_types.h"
+#include "socket_base/socket.h"
+
+#include "rfc5444/rfc5444_writer.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/** @brief Definition for RIOTs debugging option */
+#define ENABLE_DEBUG (0)
+#include "debug.h"
+
+/**
+ * @name    NHDP protocol macros
+ *
+ * @{
+ */
+#ifndef MANET_PORT
+/** @brief Well-known MANET port from RFC 5498 */
+#define MANET_PORT                  (269)
```

Comment:
I would be in favor of defining this at a more common place somewhere in `sys/net/include`, so that AODVv2 and NDHP can use the same define. Not sure, what would be the best place for it. Maybe we should introduce a new file for that similar to `sys/include/net/ng_protnum.h`.